<launch>
  <arg name="team" default="1"/>
 
  <node pkg="rosaria" name="robot_$(arg team)" type="RosAria">
	<param name="port" value="/dev/pioneer"/>
  </node>

  <node pkg="usb_cam" name="camera_$(arg team)" type="usb_cam_node">
	<param name="video_device" value="/dev/video1"/>
	<remap from="/camera/image_raw" to="/camera_$(arg team)/image_raw"/>
<!--	<param name="image_width" value="1024"/>
	<param name="image_height" value="768"/>
	<param name="autofocus" value="true"/>-->
  </node>

   <include file="$(find rplidar_ros)/launch/view_rplidar.launch" />
  <param name="rplidar/serial_port" value="/dev/rplidar"/>

<!--  <node name="rplidarNode_$(arg team)"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/rplidar"/>  
  <param name="serial_baudrate"     type="int"    value="115200"/>
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  </node>-->

  <node pkg="robohockey" name="ki_$(arg team)" type="robohockey_node" output="screen">
	<remap from="/cmd_vel" to="/robot_$(arg team)/cmd_vel"/>
	<remap from="/camera/image_raw" to="/camera_$(arg team)/image_raw"/>
  </node>

</launch>
